CHANGELOG.md

```diff
@@ -1,3 +1,31 @@
+### main
+
+#### ⚠️ Breaking change
+
+`Map<String?, Object?>?` is replaced wiht `Point` from [turf](https://pub.dev/packages/turf) package.
```

Comment:
```suggestion
`Map<String?, Object?>?` is replaced with `Point` from [turf](https://pub.dev/packages/turf) package.
```

Reply:
Can we specify link to the `Point` class https://pub.dev/documentation/turf/latest/turf/Point-class.html